laravel/app/Http/Controllers/API/CsvController.php

```diff
@@ -36,20 +37,32 @@ public function temporary_ship(Request $request)
         $inventories = Inventory::TemporaryShipSearchByStock($item_ids)->get();
 
         $ship_arranged = \Config::get('const.Constant.ship_arranged');
-        $temporary_ship_id = \Config::get('const.Constant.temporary_ship_id');
+        $temporary_flag = \Config::get('const.Constant.temporary_flag');
+        $done = \Config::get('const.Constant.done');
 
         //TODO Transaction設置予定（仮）
 
         foreach ($inventories as $inventory) {
 
-            $inventory->order_item_id = $temporary_ship_id;
+            \DB::table('order_items')->insert([
+                'order_id' => $order_id,
+                'item_code' => $inventory->item_code,
+                'ship_date' => $ship_date,
+                'weight' => $inventory->weight,
+                'temporary_flag' => $temporary_flag,
+                'done_flag' => $done
```

Comment:
定数を定義しようと考えられたのはとても良いですが、このdone_flagは型がbooleanなのでわざわざ定数を定義せずに下記でOKです！
(temporary_flagも同様）
```suggestion
                'done_flag' => true
```